www/app/Console/Commands/CleanupOrphanedProcesses.php

```
@@ -0,0 +1,197 @@
+        'schedule:work',        // Laravel scheduler
+        'schedule:run',
+        '@playwright/mcp',      // Active Playwright MCP server (Claude panel tool — in-use)
+        'pocket-dev',           // Any pocket-dev infrastructure process
+        'supervisord',
+    ];
+
+    public function handle(): int
+    {
+        $minAge   = (int) $this->option('min-age');
+        $dryRun   = (bool) $this->option('dry-run');
+        $now      = time();
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$now'. (undefined)

(UnusedLocalVariable)